core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -323,6 +346,90 @@ func getRouterAccountConfig(routerProgramAddress string) chainwriter.PDALookups
 	}
 }
 
+func offrampAddressConstant(offrampProgramAddress string) chainwriter.AccountConstant {
+	return chainwriter.AccountConstant{
```

Comment:
nit: couldn't this just be addressConstant? i.e. this isn't specific to the offramp since we pass in an arbitrary string address as param